Source/Core/Core/HW/EXI_DeviceEthernet.cpp

```diff
@@ -358,6 +358,12 @@ void CEXIETHERNET::MXCommandHandler(u32 data, u32 size)
 		data &= (data & 0xff) ^ 0xff;
 		goto write_to_register;
 
+	case BBA_TXFIFOCNT:
```

Comment:
I think the cost of the check would be negligable...but anyways it could be nice to at least put a comment there stating that it doesn't handle the other cases, just in case it causes problems with something in the future.